static/app/components/assigneeSelectorDropdown.tsx

```diff
@@ -558,11 +558,10 @@ export default function AssigneeSelectorDropdown({
   );
 
   return (
-    <AssigneeWrapper>
+    <AssigneeWrapper className={className}>
       <CompactSelect
         searchable
         clearable
-        className={className}
```

Comment:
This seems like a risky change that might break other uses of `styled(AssigneeSelectorDropdown)` in the codebase. Did you search for these?

Reply:
I did search and everything seemed fine, but I am also nervous about breaking things 😬 Do you have another idea for how to left-align the `AssigneeSelectorDropdown`?

Reply:
Update: I did something similar to what I did for `IssueCell` where I exported the `AssigneeWrapper`
https://github.com/getsentry/sentry/blob/ab03cf240427c273404d582c2778ef24c55e3382/static/app/components/assigneeSelectorDropdown.tsx#L589
so I could apply styling to it
https://github.com/getsentry/sentry/blob/ab03cf240427c273404d582c2778ef24c55e3382/static/app/views/detectors/components/issuesList.tsx#L70-L73